### ISSUE TITLE
corpora with multivalued attributes cause exceptions

### ISSUE THREAD

**Issue by vidiecan** [ufal/lindat-kontext#20]
```
 in get_attr_hierarchy(vals=[{'v': u'tisk'}, {'v': u'tisk/periodi\u010dno'}, {'v': u'tisk/periodi\u010dno/\u010dasopis'}, {'v': u'tisk/knji\u017eno'}, {'v': u'tisk/knji\u017eno/strokovno'}, {'v': u'tisk/periodi\u010dno/revija'}, {'v': u'tisk/drugo'}, {'v': u'tisk/knji\u017eno/leposlovno'}, {'v': u'internet'}], hsep=u'/', multisep=u',')
    451     values = set([])
    452     for v in vals:
=>  453         values.add(v)
    454     for value in sorted(values):
    455         level = result
values = set([]), values.add = <built-in method add of set object>, v = {'v': u'tisk'}
<type 'exceptions.TypeError'>: unhashable type: 'dict' 
      args = ("unhashable type: 'dict'",) 
      message = "unhashable type: 'dict'"
```

**Comment by vidiecan** [comment]
quick fix (hack)
```
--- a/lib/corplib.py
+++ b/lib/corplib.py
@@ -450,6 +450,8 @@ def get_attr_hierarchy(vals, hsep, multisep):
     result = {}
     values = set([])
     for v in vals:
+        if isinstance(v, dict):
+           continue
         values.add(v)
 ```

**Comment by vidiecan** [comment]
(seems) fixed in 0.9
